### PR TITLE
sublevels drop chosen

### PR DESCRIPTION
**Pull request by PablitoGS**
Sublevels in chosen drop menu and buttons fixed on mobile please review @nguasch

### CODE REVIEW

File: styles/components/pulldown-chosen.scss [hunk 1]
```diff
@@ -501,11 +501,19 @@ $CHOSEN TOGGLE
     width: 230px;
 
     li {
-      @include font-base-line;
       position: relative;
       padding: $padding $padding*2;
+      font-size: $base-font-size;
+      line-height: $base-font-size;
```
Comment (nguasch) [inline styles/components/pulldown-chosen.scss]: better put line-height to 1

File: styles/elements/_buttons.scss [hunk 2]
```diff
@@ -423,15 +425,29 @@ Styleguide #{$sgi-buttons-variants}
...
   }
 
+  &.btn-justify-icon-end {
+
+    li {
+      @include flex(2 1 auto);
+
+      &:last-of-type {
+        @include flex(1 1 0);
+      }
```
Comment (nguasch) [inline styles/elements/_buttons.scss:446]: change flex to flex-grow: 0;

**Review by nguasch — COMMENTED** [review]
please, review comments @PablitoGS

**Comment by PablitoGS** [comment]
done @nguasch

**Comment by nguasch** [comment]
👍

**Pull request merged**